docker-compose.mcp.yml

```
@@ -0,0 +1,79 @@
+      db:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      mcp-server:
+        condition: service_healthy
+    volumes:
+      # Mount MCP configuration
+      - ./config/mcp_server.yml:/app/config/mcp_server.yml:ro
+      - ./docs/MCP_INTEGRATION.md:/app/docs/MCP_INTEGRATION.md:ro
+
+  # Update worker to include MCP configuration  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)